bulk-ingester/src/test/java/org/alfresco/hxi_connector/bulk_ingester/repository/filter/AncestorFilterApplierTest.java

```diff
@@ -173,6 +177,7 @@ void shouldFilterOutWhenNextAncestorInDenied()
         given(mockPath.allow()).willReturn(emptyList());
         given(mockPath.deny()).willReturn(List.of(DENIED_NODE_REF));
         given(mockAlfrescoNode.getId()).willReturn(Long.valueOf(CHILD_ID));
+        given(mockAlfrescoNode.getNodeRef()).willReturn(NODE_REF);
```

Comment:
Maybe add a test where `NODE_REF` is in the allow or deny list?

Reply:
I see there's an IT for this later, but it might be good to have a unit test too.

Reply:
You're completely right - I added a few unit tests.